operator/roles/csi-scale/templates/csi-plugin-snapshotter.yaml.j2

```diff
@@ -47,6 +47,7 @@ spec:
           args:
             - "--csi-address=$(ADDRESS)"
             - "--v=5" # Debugging
+            - "--leader-election=false"
```

Comment:
Any specific reason to add this explicitly?

Reply:
This is being refereed from https://github.com/kubernetes-csi/external-snapshotter/blob/release-4.0/deploy/kubernetes/csi-snapshotter/setup-csi-snapshotter.yaml#L91 for csi-snapshotter sidecar pod.